plugin/pkg/admission/storage/persistentvolume/resize/admission.go

```diff
@@ -99,7 +99,7 @@ func (pvcr *persistentVolumeClaimResize) Validate(ctx context.Context, a admissi
 	}
 
 	if oldPvc.Status.Phase != api.ClaimBound {
-		return admission.NewForbidden(a, fmt.Errorf("Only bound persistent volume claims can be expanded"))
+		return admission.NewForbidden(a, fmt.Errorf("only bound persistent volume claims can be expanded"))
```

Comment:
@neolit123 
the error information is from here,
but I do not understand why this change produced an error.

Reply:
it's possibly this, like i mentioned earlier:
> maybe the unit tests try to match exact string for the errors and now this PR has changed it?

Reply:
Yes,you are right.
I find the string and I changed it.